spec/requests/sessions_controller_spec.rb

```
@@ -535,6 +535,7 @@ def create_admin_user(organization)
 
       before(:all) do
         create
+        Cartodb::Central.stubs(:sync_data_with_cartodb_central?).returns(false)
```

Same as above, Sequel 3 ignoring errors.